src/routers/UserRouter.ts

```diff
@@ -101,10 +107,16 @@ const getProfileRouter = (basePath: string = '/', routePath: string = '/users/pr
 
       const firstName = get(req.body, 'firstName', user?.given_name);
       const lastName = get(req.body, 'lastName', user?.family_name);
+      const country = get(req.body, 'country', user?.user_metadata?.country);
+      const institution = get(req.body, 'institution', user?.user_metadata?.institution);
 
       const update = {
         given_name: firstName,
         family_name: lastName,
+        user_metadata: {
+          country,
+          institution,
+        },
```

Comment:
We need to double check this won't override any other `user_metadata` fields that might be stored.

```
user_metadata: {
    ...user_metadata,
    country,
    institution
}
```
It depends how Auth0 handles individual field updates, I assume it probably does a merge of objects, and above is not required.